testing/internal/e2e/boundary/scopes.go

```diff
@@ -25,21 +25,47 @@ func CreateNewOrgApi(t testing.TB, ctx context.Context, client *api.Client) stri
 	return newOrgResult.Item.Id
 }
 
+// CreateNewProjectApi creates a new project in boundary using the go api.
+// Returns the id of the new project.
+func CreateNewProjectApi(t testing.TB, ctx context.Context, client *api.Client, scopeId string) string {
+	scopeClient := scopes.NewClient(client)
+	newOrgResult, err := scopeClient.Create(ctx, scopeId, scopes.WithName("e2e Automated Test Project"))
```

Comment:
This variable name is misleading
```suggestion
	newProjResult, err := scopeClient.Create(ctx, orgScopeId, scopes.WithName("e2e Automated Test Project"))
```